dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java

```diff
@@ -54,11 +54,11 @@ public IndexPageServlet(String contextPath, String indexPagePath) {
         }
 
         ImmutableMap<String, String> templateContext = ImmutableMap.of(BASE_URL, slashedContextPath);
-        File file = new File(indexPagePath);
-        if (file.exists()) {
-            indexPage = new FileSystemIndexPage(templateContext, file);
+        Optional<URL> indexPageResource = tryGetResource(indexPagePath);
```

Comment:
The main use case for loading file system first is to allow apps supporting custom index page, if we reverse the order,  we lose that flexibility